dev/ecf/run.nrt_ecflow.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
The commands in this script should be in a README.  Running this script is like setting up a NRT run every time.
Instead, use `ecflow_client` calls once the suite has been created by `setup_ecf.sh`.